src/core/abr/representation_chooser.ts

```diff
@@ -280,6 +296,31 @@ export default class RepresentationChooser {
   private estimator : BandwidthEstimator;
   private _initialBitrate : number;
 
+  /**
+   * Information about frames (total decoded and dropped)
+   * Emitted at each clock event.
+   */
+  private lastDroppedFrames: number = 0;
+  private lastTotalFrames: number = 0;
+  private currentDroppedFrames: number = 0;
+  private currentTotalFrames: number = 0;
```

Comment:
I'm not sure that the previous twos are needed as state of our class.
They are calculated one time and needed at the exact same moment after it. Not sure why attaching them to the class makes sense.